packages/core/package.json

```diff
@@ -0,0 +1,148 @@
+    "@blueprintjs/core": "^3.18.0",
+    "@blueprintjs/icons": "^3.18.0",
+    "axios": "^0.18.0",
+    "babel-eslint": "^10.0.3",
+    "babel-loader": "^8.0.6",
+    "bcrypt-nodejs": "0.0.3",
+    "body-parser": "^1.18.2",
+    "buildmail": "^4.0.1",
+    "chalk": "^2.3.0",
+    "chokidar": "^3.0.2",
+    "compression": "^1.7.1",
+    "cookie-parser": "^1.4.3",
+    "cookie-session": "^2.0.0-beta.3",
+    "core-js": "^2.6.9",
+    "css-loader": "^3.2.0",
+    "css2json": "^1.0.0",
+    "d3-array": "^1.2.1",
+    "d3-selection": "^1.4.1",
+    "d3plus-react": "^0.5.3",
+    "d3plus-text": "^0.9.50",
+    "d3plus-viz": "^0.12.41",
+    "dom4": "^2.1.5",
+    "es6-promise": "^4.1.0",
+    "express": "^4.16.2",
+    "file-loader": "^4.2.0",
+    "hard-source-webpack-plugin": "^0.13.1",
+    "helmet": "^3.9.0",
+    "history": "^4.10.1",
+    "i18next": "^10.0.7",
+    "i18next-express-middleware": "^1.7.0",
+    "i18next-node-fs-backend": "^2.1.1",
+    "i18next-scanner": "^2.2.1",
+    "inline-environment-variables-webpack-plugin": "^1.2.1",
+    "iso-morphic-style-loader": "^2.0.1",
+    "js-yaml": "^3.10.0",
+    "lost": "^8.2.0",
+    "mailgun-js": "^0.13.1",
+    "mini-css-extract-plugin": "^0.8.0",
+    "normalize.css": "^7.0.0",
+    "opbeat": "^4.17.0",
+    "passport": "^0.4.0",
+    "passport-facebook": "^2.1.1",
+    "passport-github": "^1.1.0",
+    "passport-google-oauth20": "^1.0.0",
+    "passport-instagram": "^1.0.0",
+    "passport-linkedin-oauth2": "^1.5.0",
+    "passport-local": "^1.0.0",
+    "passport-twitter": "^1.0.4",
+    "pg": "^6.4.2",
+    "pixrem": "^5.0.0",
+    "postcss-color-function": "^4.1.0",
+    "postcss-conditionals": "^2.1.0",
+    "postcss-css-variables": "^0.13.0",
+    "postcss-each": "^0.10.0",
+    "postcss-flexbugs-fixes": "^4.1.0",
+    "postcss-for": "^2.1.1",
+    "postcss-import": "^12.0.1",
+    "postcss-loader": "^3.0.0",
+    "postcss-map": "^0.11.0",
+    "postcss-mixins": "^6.2.2",
+    "postcss-preset-env": "^6.7.0",
+    "postcss-reporter": "^6.0.1",
+    "postcss-url": "^8.0.0",
+    "pretty": "^2.0.0",
+    "promise-throttle": "^1.0.0",
+    "prop-types": "^15.6.0",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-helmet": "^5.2.0",
+    "react-hot-loader": "^4.12.11",
+    "react-i18next": "^9.0.1",
+    "react-redux": "^5.1.1",
+    "react-router": "3.2.3",
+    "react-router-redux": "^4.0.8",
+    "react-scroll": "^1.7.13",
+    "redux": "^4.0.4",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.0.1",
+    "sequelize": "^4.13.10",
+    "serialize-javascript": "^1.4.0",
+    "shelljs": "^0.8.3",
+    "url-loader": "^2.1.0",
+    "urllite": "^0.5.0",
+    "webpack": "4.38.0",
+    "webpack-bundle-analyzer": "^3.4.1",
+    "webpack-cli": "^3.3.7",
+    "webpack-dev-middleware": "^3.7.0",
+    "webpack-hot-middleware": "^2.25.0",
+    "webpackbar": "^4.0.0",
+    "yml-loader": "^2.1.0",
+    "yn": "^3.0.0"
+  },
+  "optionalDependencies": {
+    "d3plus-react": "^0.5.9"
```

Comment:
this got added to `optionalDependencies`, but is still listed in `dependencies`?

Reply:
oh, dang
I might missed that on the combination.